### ISSUE TITLE
Tray Icon Signature changes with every update

### ISSUE THREAD

**Issue by OzanKurt** [HearthSim/Hearthstone-Deck-Tracker#3632]
![Image](https://user-images.githubusercontent.com/3404787/38762442-52c7d172-3f8c-11e8-812d-be1020fe73fe.png)

You can not imagine how annoying it is to do the same thing over and over again for each update.

**Comment by judge2020** [comment]
Relevant squirrel issue: https://github.com/Squirrel/Squirrel.Windows/issues/863, I'm not sure if this is something fixable in HDT itself.

**Comment by OzanKurt** [comment]
So what can we do to fix this?

**Comment by azeier** [comment]
Actually - I'm not sure whether this fixes it, but I removed the version number from the icon text for now. I suppose there is a chance of that being the problem.

**Comment by OzanKurt** [comment]
Yeah that might have been the case. Windows is retarded enough to mark them as different applications. :+1:

**Comment by azeier** [comment]
Renaming the exe has all sorts of implications, definitely can't do that.

**Comment by OzanKurt** [comment]
![image](https://user-images.githubusercontent.com/8682003/38883991-b11d5dde-426e-11e8-945d-93382ae2d1f2.png)

It seems to work now with the new update!

**Comment by OzanKurt** [comment]
![image](https://user-images.githubusercontent.com/8682003/39402119-6845125e-4b57-11e8-8726-f5bf486b8c72.png)

We still have the same problem even if the signature is same. ;/

**Comment by riQQ** [comment]
@OzanKurt does this work now?
